src/components/user/uc/users.uc.js

```diff
@@ -46,9 +46,11 @@ const deleteUserRelatedData = async (userId, schoolTombstoneUserId, deleteUserFa
 		for (const deleteFn of facade.deleteUserData) {
 			try {
 				// eslint-disable-next-line no-await-in-loop
-				const trash = await deleteFn(userId, schoolTombstoneUserId);
-				// eslint-disable-next-line no-await-in-loop
-				await trashbinRepo.updateTrashbinByUserId(userId, trash.data);
+				const trash = await deleteFn(userId);
+				if (trash.trashBinData.data.length > 0) {
+					// eslint-disable-next-line no-await-in-loop
+					await trashbinRepo.updateTrashbinByUserId(userId, [trash.trashBinData]);
+				}
 			} catch (error) {
 				errorUtils.asyncErrorLog(error, `failed to delete user data for facade ${facadeName}#${deleteFn.name}`);
```

Comment:
i would suggest when we reach the catch clause here to keep a notice in the users trashbin. to not only have this information in the log without trashbindata added by scope.